internal/cmd/commands/server/server.go

```diff
@@ -667,69 +667,100 @@ func (c *Command) StartWorker() error {
 
 func (c *Command) WaitForInterrupt() int {
 	const op = "server.(Command).WaitForInterrupt"
-	// Wait for shutdown
-	shutdownTriggered := false
 
-	// Add a force-shutdown goroutine to consume another interrupt
-	abortForceShutdownCh := make(chan struct{})
-	defer close(abortForceShutdownCh)
+	var shutdownCompleted atm.Bool
+	shutdownCompleted.Store(false)
```

Comment:
The zero value is false per the docs so you don't need the Store(false) here or below